coral/src/app/features/topics/details/subscriptions/TopicSubscriptions.tsx

```diff
@@ -250,6 +259,7 @@ const TopicSubscriptions = () => {
         ]}
         table={
           <TopicSubscriptionsTable
+            isLoading={topicOverviewIsRefetching}
```

Comment:
I would avoid `isLoading` as a prop name for our own components to dissociate clearly from the `react-query` `isLoading`. Maybe `isUpdating` ? Or even `isRefetching`.

Reply:
I think `isUpdating` is a good choice. It describes better that this is not the initial loading but does not introduce tanstack specific language!